traitsui/instance_choice.py

```diff
@@ -12,14 +12,16 @@
     instance editor factory classes.
 """
 
+from abc import abstractmethod
+
 from traits.api import HasPrivateTraits, Str, Any, Dict, Tuple, Callable, Bool
 
 from .ui_traits import AView
 
 from .helper import user_name_for
 
 
-class InstanceChoiceItem(HasPrivateTraits):
+class InstanceChoiceItem(ABCHasStrictTraits):
```

Comment:
You'll have to update the `traits.api` import statement to include `ABCHasStrictTraits`

Reply:
Done.